redaxo/src/core/lib/sql/column.php

```diff
@@ -0,0 +1,170 @@
+<?php
+
+/**
+ * Class to represent sql columns.
+ *
+ * @author gharlan
+ *
+ * @package redaxo\core
+ */
+class rex_sql_column
+{
+    private $name;
+    private $type;
+    private $nullable;
+    private $default;
+    private $extra;
+
+    private $modified = false;
+
+    /**
+     * @param string      $name
+     * @param string      $type
+     * @param bool        $nullable
+     * @param null|string $default
+     * @param null|string $extra
+     */
+    public function __construct($name, $type, $nullable = false, $default = null, $extra = null)
+    {
+        $this->name = $name;
+        $this->type = $type;
+        $this->nullable = $nullable;
+        $this->default = $default;
+        $this->extra = $extra;
+    }
+
+    /**
+     * @param bool $modified
+     *
+     * @return $this|rex_sql_column
+     */
+    public function setModified($modified)
+    {
+        $this->modified = $modified;
+
+        return $this;
+    }
+
+    /**
+     * @return bool
+     */
+    public function isModified()
+    {
+        return $this->modified;
+    }
+
+    /**
+     * @param string $name
+     *
+     * @return $this|rex_sql_column
```

Comment:
Total ätzend, und ist schon lange so, und gibt auch Issues dazu in deren Bugtracker.